build/build-parent/pom.xml

```diff
@@ -651,7 +651,8 @@
     <profile>
       <id>sonarcloud-analysis</id>
       <properties>
-        <sonar.coverage.jacoco.xmlReportPaths>${project.reporting.outputDirectory}/jacoco/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
+        <sonar.coverage.jacoco.xmlReportPaths>${project.reporting.outputDirectory}/jacoco/jacoco.xml
+        </sonar.coverage.jacoco.xmlReportPaths>
```

Comment:
Unrelated change?

Reply:
Unrelated change; from IDEA XML formatter. (/me learns one can use "format only changed lines" instead of "format whole file" as a save action).